### ISSUE TITLE
Refactor menu items to be semantically correct

### ISSUE THREAD

**Issue by joonasmkauppinen** [joonasmkauppinen/yle-campus-lunch-list#11]
Menu items should be rendered as unordered list, not as paragraphs.
